common/tools/field_test_log_player/README.md

```diff
@@ -47,4 +47,4 @@ Active keys in plotter view:
 
     spacebar    - pause/resume, zooming possible during pause
     +           - skip next 30seconds in log file
-    -           - rewind 30seconds in log file
\ No newline at end of file
+    -           - rewind 30seconds in log file
```

Comment:
Why this change?

Reply:
Correct, this is unnecessary line. Removed.